Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Tools/IDE/Wing/readme.md

```diff
@@ -57,7 +100,61 @@ The DCCsi integrations are workflow integrations with the intent to provide an i
+
+- Supports O3DE Python as Launch Configuration:
+  
+  - `${DCCSI_PY_BASE}`
+
+- Supports Blender Python as a Launch Configuration:
+  
+  - `${DCCSI_PY_BLENDER}`
+
+- Supports MayaPy as a Launch Configuration:
+  
+  - `${DCCSI_PY_MAYA}`
+
+- Supports Substance3D Designer Python as a Launch Configuration:
+  
+  - `${DCCSI_PY_SUBSTANCE}`
+
+You can change the current Launch environment a couple of differenty ways:
+1. Globally, access the 'Launch Configuations' via the following menu:
+    - `Wing > Python Shell (tab) > options > Use Environment > (select a launch environment)`
+
+2. On Script Executtion:
+    - press the green Start button in the upper-right of the main toolbar ![image](https://user-images.githubusercontent.com/23222931/201775116-1030cf5b-9749-464e-9595-fbe7c09839cd.png)
+    - or the F5 button
+    - In the dialog box:
+        - set the primary pull down to `Use Selected Launch Configuration`
+        - set the second pulldown to the interpretter you'd like to run
+
+![image](https://user-images.githubusercontent.com/23222931/201775372-861a36e7-66c5-4b22-80e1-7b764c68d6c8.png)
+    
+#### Latest Features:
+
+- The latest version of Wing Pro 8 (8.3.3.1) is recommended, as there was a bug in the initial release that prevented Launch Configurations bound to a ENVAR (like above) to function properly. Wing Pro 9 likely works as well, you may just have to make slight adjustments to the configuration (covered in this document below)
+
+- The DccScriptingInterface Gem (DCCsi) can be added to your Game Project. See [Registering Gems to a Project](https://www.o3de.org/docs/user-guide/project-config/register-gems/), and [Adding and Removing Gems in a Project](https://www.o3de.org/docs/user-guide/project-config/add-remove-gems/) which you can do through the Project Manager (o3de.exe) With the DCCsi enabled, you can launch Wing Pro 8 via menus in the main Editor.
+
+- Some additional testing and improvement have been mad in the current release (2210), including changes to better support Installer build folder patterns. See the [readme.md at the root of the DCCsi](https://github.com/o3de/o3de/blob/development/Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/readme.md) for more information about advanced configuration.
+
+# Additional Information
 
 Each IDE is different.  Some TAs (like me) like Wing IDE because it's relatively easy to set up and works great with DCC apps like Maya with less configuration.  It's also pretty powerful and you can do a lot of data-driven configuration with it (which we will be covering some of.)  We aren't picking an IDE for you, that's a personal choice, we are just showing you how one like Wing can be set up to provide a better out-of-box experience (so that you don't have to do the setup and configuration yourself.)  In fact, we intend to do the same for other IDS like PyCharm and VScode (maybe others in the future.)
```

Comment:
To me this should go higher, explain that Wing IDE is not proscriptive and that other IDEs are available.

I would also avoid personal pronouns as its very unclear who "like me" referes to here

Reply:
gotcha, will make changes

Reply:
@lmbr-pip see if you like the current changes better?